### PR TITLE
Fixes for the songs for MAP20 and DM26

### PR DESCRIPTION
**Pull request by Josephus-DH4050-Astartes**

### CODE REVIEW

**Comment by mc776** [comment]
What is being fixed here?

**Comment by Josephus-DH4050-Astartes** [comment]
> What is being fixed here?

Some parts of the songs that I wasn't really proud of.

**Review by mc776 — APPROVED** [review]

**Comment by mc776** [comment]
I'll remove the "bug" tag since it's an aesthetic issue rather than something technical like a long pause between loops or a melodic line being inaudible in OPL or something, but it's approved.

I'm leaving it unmerged for a bit just in case anything comes up though.

**Comment by Josephus-DH4050-Astartes** [comment]
> I'll remove the "bug" tag since it's an aesthetic issue rather than something technical like a long pause between loops or a melodic line being inaudible in OPL or something, but it's approved.
> 
> I'm leaving it unmerged for a bit just in case anything comes up though.

Awesome!

**Comment by mc776** [comment]
Just gave this an actual listen. I like the changes!

But

the DM26 music file needs a d_ in front of the name

File: musics/dm26.mid
Comment (mc776) [inline musics/dm26.mid:1]: needs to be renamed to d_dm26.mid

**Comment by Josephus-DH4050-Astartes** [comment]
> Just gave this an actual listen. I like the changes!
> 
> But
> 
> the DM26 music file needs a d_ in front of the name

no problem! ^^

**Review by mc776 — APPROVED** [review]

**Pull request merged**